app/controllers/users.js

```diff
@@ -1,21 +1,24 @@
-const { createUser, findUser, getUsers } = require('../services/users');
+const { createUser, findUser, getUsers, updateUser } = require('../services/users');
 const lodash = require('lodash');
 const logger = require('../logger');
 const { validatePassword } = require('../utils/helpers');
 const errors = require('../errors');
-const { PAGESIZE, FIRSTPAGE } = require('../constants');
+const { PAGESIZE, FIRSTPAGE, ROLES } = require('../constants');
+
+const pickFromObject = (object, params = ['name', 'last_name', 'email', 'role']) =>
```

Comment:
object? wouldnt be user?

Reply:
But what happens if i need to use that function but for a non-user object?